.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = https://github.com/ceph/ceph-object-corpus.git
 [submodule "src/civetweb"]
 	path = src/civetweb
-	url = https://github.com/ceph/civetweb
+	url = https://github.com/mdw-at-linuxbox/civetweb
```

Comment:
hmm, well as you specify, we can't merge this--is there civetweb change proposed in a PR as well?